app/src/main/java/com/example/pinterbest/api/ApiClient.kt

```diff
@@ -9,6 +9,31 @@ import retrofit2.Retrofit
 import retrofit2.converter.gson.GsonConverterFactory
 
 class ApiClient {
+    private var mInstance: ApiClient? = null
+    private var mRetrofit: Retrofit
+
+    init {
+        mRetrofit = Retrofit.Builder()
```

Comment:
you can avoid `init` and make mRetrofit va**L** if rewrite:
```kotlin
private val mRetrofit: Retrofit = Retrofit.Builder()
            .baseUrl(BASE_URL)
             .addConverterFactory(GsonConverterFactory.create())
             .client(okHttpClient().build())
             .build()
```

Reply:
Fixed here https://github.com/enginire4ek/Pinterbest/pull/3/commits/5fc6193ab1b0ed6d7b799a8929f6e95603658369